.changelog/2509.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:enhancement
+resource/mongodbatlas_encryption_at_rest: Adds new `azure_key_vault_config.#.require_private_networking` field to enable connection to Azure Key Vault over private networking
+```
```

Comment:
is docs update for this resource handled in another ticket? Just to make sure we don't forget.

Reply:
yes docs for this will be updated to be auto-generated in another ticket